README.md

```diff
@@ -1,29 +1,24 @@
 # DynamicParametersBundle
 
-This bundle provides a way to read parameters from environment variables at runtime.
-The value defined in the container parameter is used as fallback when the environment variable is not available.
+Inspired by [incenteev/dynamic-parameters-bundle](https://github.com/Incenteev/DynamicParametersBundle) and [%env()%](http://symfony.com/blog/new-in-symfony-3-2-runtime-environment-variables) parameters in Symfony 3.2.
```

Comment:
I'd rephrase this: "Bundle is fork of `incenteev/dynamic-parameters-bundle` which is abandoned.